src/app/modules/project/add/source/source.component.ts

```diff
@@ -45,4 +53,27 @@ export class SourceComponent implements OnInit {
       this.mostUsedSources.push(demoSource);
     }
   }
+
+  /**
+   * Method which triggers when the submit source uri button is pressed.
+   * Fetches the source from the wizard service.
+   */
+  public onClickSubmitSourceUri(): void {
+    const sourceUri = this.sourceUriInput.value;
+    if (sourceUri == null || sourceUri === '') {
+      // TODO: display error invalid uri
```

Comment:
TODO :)

Reply:
Error handling will be implemented in a later PR when the designs for throwing errors are complete